tlsdefaults.go

```diff
@@ -11,9 +11,7 @@ var (
 	preferredCipherSuites = []uint16{
 		tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
 		tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
-		tls.TLS_ECDHE_RSA_WITH_RC4_128_SHA,
 		tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,
-		tls.TLS_RSA_WITH_RC4_128_SHA,
```

Comment:
Oh nice! I didn't find that. Yeah we should just use the defaults then @oxtoacart